jobs/postfix/templates/bin/ctl

```diff
@@ -11,12 +11,11 @@ case $1 in
 
   start)
       /etc/init.d/postfix start
-
       ;;
 
   stop)
       /etc/init.d/postfix stop
-
+      sleep 5
```

Comment:
Why is the sleep necessary in a control script?

Reply:
it was left over from a test.  I will remove it!